packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -79,6 +79,10 @@ const propTypes = {
    * Ref callback function to be applied to content container.
    */
   refCallback: PropTypes.func,
+  /**
+   * Header Title for menu if menu contains drill-in sub-menus
+   */
+  headerTitle: PropTypes.string,
```

Comment:
maybe, add a default value to this prop?

Reply:
Default value is defined here [Menu](https://github.com/cerner/terra-framework/blob/814b2bf3b5e65676e29562789da76819cefba73b/packages/terra-menu/src/Menu.jsx#L67)

Reply:
We should define a default here as well

Reply:
Updated here 277939ef97fccdb445d4a11850b83f430e56c73b